pom.xml

```diff
@@ -306,6 +306,23 @@
           <artifactId>maven-resources-plugin</artifactId>
           <version>3.2.0</version>
         </plugin>
+        <plugin>
+          <!-- Copied from com.google.cloud:google-cloud-shared-config:0.10.0 -->
+          <groupId>com.coveo</groupId>
+          <artifactId>fmt-maven-plugin</artifactId>
+          <version>2.9</version>
+          <dependencies>
+            <dependency>
+              <groupId>com.google.googlejavaformat</groupId>
+              <artifactId>google-java-format</artifactId>
+              <version>1.7</version>
+            </dependency>
+          </dependencies>
+          <configuration>
+            <style>google</style>
+            <verbose>true</verbose>
+          </configuration>
+        </plugin>
```

Comment:
This is not working.

```
 Failed to execute goal com.coveo:fmt-maven-plugin:2.10:check (default-cli) on project google-api-client-parent: 
Execution default-cli of goal com.coveo:fmt-maven-plugin:2.10:check failed: An API incompatibility was 
encountered while executing com.coveo:fmt-maven-plugin:2.10:check: java.lang.UnsupportedClassVersionError: 
com/google/googlejavaformat/java/FormatterException has been compiled by a more recent version of the Java 
Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```

Reply:
There's another place that specify the newer version of plugin and google-java-format dependency: https://github.com/googleapis/google-api-java-client/blob/master/pom.xml#L460